e2e_test/source/cdc/cdc.share_stream.slt

```diff
@@ -268,3 +268,62 @@ SELECT * from person_new order by id;
 1100 noris ypl@qbxfg.com 1864 2539 enne
 1101 white myc@xpmpe.com 8157 6974 se
 1102 spencer wip@dkaap.com 9481 6270 angeles
+
+statement ok
+CREATE TABLE numeric_to_rw_int256_shared (
+    id int,
+    num rw_int256,
+    PRIMARY KEY (id)
+) FROM pg_source TABLE 'public.numeric_table';
+
+statement ok
+CREATE TABLE numeric_to_varchar_shared (
+    id int,
+    num varchar,
+    PRIMARY KEY (id)
+) FROM pg_source TABLE 'public.numeric_table';
+
+
+system ok
+psql -c "
+insert into numeric_table values(102, 57896044618658097711785492504343953926634992332820282019728792003956564819967);
+--- 2^255
+insert into numeric_table values(103, 57896044618658097711785492504343953926634992332820282019728792003956564819968);
+--- 2^256
+insert into numeric_table values(104, 115792089237316195423570985008687907853269984665640564039457584007913129639936);
+insert into numeric_table values(105, 115792089237316195423570985008687907853269984665640564039457584007913129639936.555555);
+"
+
+sleep 3s
+
+query III rowsort
+select * from numeric_to_varchar_shared
+----
+1   3.14
+102 57896044618658097711785492504343953926634992332820282019728792003956564819967
+103 57896044618658097711785492504343953926634992332820282019728792003956564819968
+104 115792089237316195423570985008687907853269984665640564039457584007913129639936
+105 115792089237316195423570985008687907853269984665640564039457584007913129639936.555555
+2   57896044618658097711785492504343953926634992332820282019728792003956564819967
+3   57896044618658097711785492504343953926634992332820282019728792003956564819968
+4   115792089237316195423570985008687907853269984665640564039457584007913129639936
+5   115792089237316195423570985008687907853269984665640564039457584007913129639936.555555
+
+# The invalid data for rw_int256 is converted to NULL
+query III rowsort
+select * from numeric_to_rw_int256_shared
+----
+1   NULL
+102 57896044618658097711785492504343953926634992332820282019728792003956564819967
+103 NULL
```

Comment:
Why 103-105 and 3-5 is `null`?

Reply:
```
# float => invalid for rw_int256
3.14 

# 2^255 - 1 => valid 
57896044618658097711785492504343953926634992332820282019728792003956564819967

# 2^255 => invalid due to overflow 
57896044618658097711785492504343953926634992332820282019728792003956564819968

# 2^256 => invalid due to overflow 
115792089237316195423570985008687907853269984665640564039457584007913129639936

# invalid due to overflow and float
115792089237316195423570985008687907853269984665640564039457584007913129639936.555555
```

Reply:
ok, previously I thought we will discard the fractional part of float.